core/templates/dev/head/expressions/expressionSyntaxTree.js

```diff
@@ -393,7 +393,9 @@ oppia.factory('expressionSyntaxTreeService', [
         eval: function(args) {
           verifyNumArgs(args, 2);
           var numericArgs = _coerceAllArgsToNumber(args);
-          return Math.log(numericArgs[0]) / Math.log(numericArgs[1]);
+          var preciseAns = Math.log(numericArgs[0]) / Math.log(numericArgs[1]);
+          // Round answer upto 9 decimal places
```

Comment:
Let's add a bit more clarity to this comment: We round answers to 9 decimal places, so that we don't run into issues like log(9, 3) = 2.0000000000004.